votor/src/consensus_pool/vote_pool.rs

```diff
@@ -162,13 +144,13 @@ impl DuplicateBlockVotePool {
             .get(validator_vote_key)
             .is_some_and(|vs| vs.contains(block_id))
     }
-}
 
-impl VotePool for DuplicateBlockVotePool {
-    fn total_stake(&self) -> Stake {
+    #[cfg(test)]
+    pub(super) fn total_stake(&self) -> Stake {
```

Comment:
Hmm, does total_stake ever make sense for DuplicateBlockVotePool? If not, we can remove total_stake() from trait and fix tests as necessary.